aspnetcore/tutorials/razor-pages/model.md

```diff
@@ -1306,7 +1306,7 @@ Examine the `Up` method.
 
 ### Test the app
 
-* Run the app and append `/Movies` to the URL in the browser (`http://localhost:port/movies`).
+* Run the app `dotnet run` and append `/Movies` to the URL in the browser (`http://localhost:port/movies`).
```

Comment:
Problems:
* This doesn't apply to Visual Studio users, 80+% of our audience.
* You've got this in a very old version of the tutorial.
* Most folks by now know how to run the app.